app/CLI/Actions.hs

```diff
@@ -221,8 +222,10 @@ handleCliAction sendRequestToDelegate currentDelegateStateRef userAction = do
               , utxoToCommit = standingBidTxIn
               }
     NewBid auctionName bidAmount -> do
-      -- FIXME: proper error printing
-      Just CliEnhancedAuctionTerms {terms, sellerActor} <- liftIO $ readCliEnhancedAuctionTerms auctionName
+      CliEnhancedAuctionTerms {terms, sellerActor} <-
```

Comment:
L1/L2 actions are still essentially copy-pasted which stands for `code duplication in new Actions`.